### ISSUE TITLE
CRITICAL: GC accounts switch up when gc resource is restarted (auto login)

### ISSUE THREAD

**Issue by AfuSensi** [MrGreenGaming/MTA-Resources#488]
On gc resource restart, autologin will log players into the wrong accounts. It also replaces a database entry in gc_autologin, so players will keep getting logged in to that account even after reconnecting.

I suspect this bug is caused by api call mixups when requesting /api/account/details for the whole server.

**Possible fixes**: onResourceStart, use /api/accounts/details-multiple for current players in server, after that, use normal /api/account/details.

Tempfix is incoming, no idea if it will work though.

**Comment by Mihoje** [comment]
This issue also causes the same player being written in the name cache database on two records with two different forum ids.

**Comment by Cena1** [comment]
It is done, right @AfuSensi ? If so close this one!

**Comment by AfuSensi** [comment]
No, still need to implement a proper fix. Right now it stops people from getting logged into other accounts, but when that would happen, the autologin just stops logging in, causing players needing to relogin/reconnect. I know what the solution is, just have not found the time to do it yet.

**Comment by VulpyWags** [comment]
Current solution is acceptable
